presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcWarnings.java

```diff
@@ -122,69 +134,47 @@ public void testStatementWarnings()
 
     @Test
     public void testLongRunningStatement()
-            throws SQLException, InterruptedException
+            throws Exception
     {
         ExecutorService queryExecutor = newSingleThreadExecutor(daemonThreadsNamed("test-%s"));
         QueryCreationFuture queryCreationFuture = new QueryCreationFuture();
         queryExecutor.submit(() -> {
             try {
-                statement.execute("CREATE SCHEMA blackhole.blackhole");
-                statement.execute("CREATE TABLE blackhole.blackhole.test_table AS SELECT 1 AS col1 FROM tpch.sf1.lineitem CROSS JOIN tpch.sf1.lineitem");
```

Comment:
The refactor of this test that replaces lineitem self CROSS JOIN with backhole slow table is not related to moving JDBC warnings to Statement and so should go in separate commit.